internal/tool/ssh/deploy.go

```diff
@@ -131,30 +133,59 @@ func deploy(ctx context.Context, args []string) error {
 	}
 }
 
+func getTmpDirs(locs []string, depId string) (map[string]string, error) {
+	tmpDirs := make(map[string]string, len(locs))
+
+	for _, loc := range locs {
+		cmd := exec.Command("ssh", loc, "mktemp", "-u")
+
+		if err := cmd.Run(); err != nil {
+			return nil, err
+		}
```

Comment:
we don't need this. Actually this leads to an error.  cmd.Output() already runs the command.